utils/build/docker/nodejs/express4/app.js

```diff
@@ -87,6 +87,22 @@ app.get("/make_distant_call", (req, res) => {
   });
 });
 
+app.get("/users", (req, res) => {
+  let user = {}
+  if (req.query['user']) {
+    user.id = req.query['user']
+  } else {
+    user.id = 'anonymous'
+  }
+
+  const ret = tracer.appsec.isUserBlocked(user)
+  if (ret) {
+    tracer.appsec.blockRequest(req, res)
+  } else {
+    res.send(`Hello ${user.id}`)
+  }
```

Comment:
Let's try to not use var names like `ret`
```suggestion
  const shouldBlock = tracer.appsec.isUserBlocked(user)
  if (shouldBlock) {
    tracer.appsec.blockRequest(req, res)
  } else {
    res.send(`Hello ${user.id}`)
  }
```